H_ecto_models.md

```diff
@@ -419,22 +405,18 @@ At this point, let's see how we can actually use Ecto in our application. Luckil
 
 Let's work through the generated controller action by action to see how Ecto is used.
 
-Before we get to the first action, let's look at two important lines at the top of the file.
+Before we get to the first action, let's look at an important line at the top of the file.
```

Comment:
I think we can replace this line with:

>  We can alias `HelloPhoenix.User` so that we can name our structs `%User{}` instead of `%HelloPhoenix.User{}`.

And delete line 418